src/libraries/System.Private.CoreLib/src/System/Numerics/Vector4.cs

```diff
@@ -209,11 +209,7 @@ public static Vector4 Clamp(Vector4 value1, Vector4 min, Vector4 max)
         [MethodImpl(MethodImplOptions.AggressiveInlining)]
         public static Vector4 Lerp(Vector4 value1, Vector4 value2, float amount)
         {
-            return new Vector4(
-                value1.X + (value2.X - value1.X) * amount,
-                value1.Y + (value2.Y - value1.Y) * amount,
-                value1.Z + (value2.Z - value1.Z) * amount,
-                value1.W + (value2.W - value1.W) * amount);
+            return (value2 - value1) * new Vector4(amount) + value1;
```

Comment:
Why not just `value1 + (value2 - value1) * amount`? It keeps the same ordering as we had originally (even if the eventual order of ops is still the same) and relies on the existing JIT optimization around `Vector4 * float`...